### PR TITLE
Fix documentation style of hdm parser.

### PR DESCRIPTION
**Pull request by MaxJPRey**

### CODE REVIEW

File: pyaedt/sbrplus/hdm_parser.py [hunk 1]
```diff
@@ -42,11 +42,11 @@ def __init__(self, filename):
         self.binarycontent = binarycontent
 
     def parse_message(self):
-        """Parses the binary content of the HDM file"""
+        """Parse the binary content of the HDM file."""
         return self._parse(self.message["type"])
 
     def _parse(self, type_name):
-        """Generic parser method, dispatches to specialized ones"""
+        """Generic parser method, dispatches to specialized ones."""
```
Comment (PipKat) [inline pyaedt/sbrplus/hdm_parser.py:49]: Method description should start with a verb. Would we be able to use this instead: "Use generic parser method, which dispatches to specialized parsers."? What are we parsing?

File: pyaedt/sbrplus/hdm_parser.py [hunk 2]
```diff
@@ -88,7 +88,7 @@ def _parse_list(self, type=None, base=None, size=1):
         """
         Parser for vector or list. 'vector's are to be interpreted in the linear algebra sense,
         and converted to numpy.array. 'list's are Python lists. Only simple base types can be
```
Comment (PipKat) [inline pyaedt/sbrplus/hdm_parser.py:90]:
```suggestion
        Parser for vector or list. A vector is interpreted in the linear algebra sense
        and converted to a NumPy array. A list is converted to a Python list. Only simple base types can be
```

File: pyaedt/sbrplus/hdm_parser.py [hunk 3]
```diff
@@ -88,7 +88,7 @@ def _parse_list(self, type=None, base=None, size=1):
...
-        interpreted as a numpy array
+        interpreted as a numpy array.
```
Comment (PipKat) [inline pyaedt/sbrplus/hdm_parser.py:91]:
```suggestion
        interpreted as a NumPy array.
```

File: pyaedt/sbrplus/hdm_parser.py (hunk 1 above)
Comment (MaxJPRey) [inline pyaedt/sbrplus/hdm_parser.py:49]:
```suggestion
        """Use a generic parser method, which dispatches to appropriate and specialized parsers."""
```

**Comment by MaxJPRey** [comment]
Thanks @PipKat for the review.

**Comment by Samuelopez-ansys** [comment]
@zebanaqviWAT @jsalant22 Could you commit the comments and merge this PR? Thanks

**Review by Samuelopez-ansys — APPROVED** [review]

**Pull request merged**